generator-theia/src/common/app-package-generator.ts

```diff
@@ -19,25 +19,30 @@ export class AppPackageGenerator extends AbstractGenerator {
     }
 
     protected compilePackage(): NodePackage {
-        const dependencies = this.isWeb() ? {} : {}
+        const dependencies = this.isWeb() ? {} : {};
```

Comment:
That line doesn't seem to make a lot of sense (I know you only added the semicolon :-))

Reply:
I've removed it